gfi/populate.py

```diff
@@ -117,13 +122,15 @@ def get_repository_info(owner, name):
         DATA = toml.load(REPO_DATA_FILE)
 
         LOGGER.info("Found %d repository entries in %s", len(DATA["repositories"]), REPO_DATA_FILE)
-
+        twitter = Twython(APP_KEY, APP_SECRET, OAUTH_TOKEN, OAUTH_TOKEN_SECRET)
         for repository_url in DATA["repositories"]:
             repo_dict = parse_github_url(repository_url)
             if repo_dict:
                 repo_details = get_repository_info(repo_dict["owner"], repo_dict["name"])
                 if repo_details:
                     REPOSITORIES.append(repo_details)
+                    tweet_string = TWEET_TEMPLATE.substitute(owner = repo_dict["owner"], url = repo_dict["url"])
```

Comment:
https://twitter.com/gfibot/status/1227011275260977153 -- Can we use this format?

If there is a way to read the project's description.

<img width="662" alt="Screenshot 2020-02-21 at 2 09 56 AM" src="https://user-images.githubusercontent.com/4883668/75025025-4aa99a00-544f-11ea-8fa9-8f4f858cf04a.png">

Reply:
Created a template and added the same with the PR. Do review and let me know if I'm missing anything.